sunflower-land/sunflower-land
# [FEAT] Offer made flow

# Description

Allow players to submit offers for:

- Collectibles
- Wearables
- Buds

<img width="529" alt="Screenshot 2024-08-28 at 12 03 33 PM" src="https://github.com/user-attachments/assets/92bdf816-8a28-49fa-8021-b273da690191">
<img width="522" alt="Screenshot 2024-08-28 at 12 03 50 PM" src="https://github.com/user-attachments/assets/58e6342b-87fe-4a58-b6b7-9f5e90cf7a12">
<img width="397" alt="Screenshot 2024-08-28 at 12 03 55 PM" src="https://github.com/user-attachments/assets/3c7fefe5-cb79-4475-9018-361b263059c4">
<img width="545" alt="Screenshot 2024-08-28 at 12 05 30 PM" src="https://github.com/user-attachments/assets/5d0bdee0-2563-4e1a-97a2-20b448dfe736">


## How to test?

For this flow you will want to test making offers for collectibles, wearables + buds. You can also test instant and onchain offers.

After making an offer, it will appear in the offer section.

1. Connect to local API
2. Open up marketplace
3. Select an item and submit an offer

_Note: The wearable collection has a weird bug when you click on a list card it takes you to the wrong place._
